### ISSUE TITLE
Please add nova_compute support for "min_count"

### ISSUE THREAD

**Issue by greenmoss** [ansible/ansible#8131]
This will allow multiple Openstack compute nodes to be launched in parallel. Reference discussion:

https://groups.google.com/d/msg/ansible-project/8eEn8l2qZPE/-y0JNg1ymw0J

**Comment by ansibot** [comment]
Thanks for filing a ticket! I am the friendly GitHub Ansibot. I see you did not fill out your issue description based on our new issue template. Please copy the contents of this [template](https://raw2.github.com/ansible/ansible/devel/examples/issues/ISSUE_TEMPLATE.md) and **paste it into the description** of your ticket. 

Our system will automatically close tickets that do not have an issue template filled out within 7 days. Please note that due to large interest in Ansible, humans may not comment on your ticket if you ask them questions. Don't worry, you're still in the queue and the robots are looking after you.

**Comment by mpdehaan** [comment]
per mailing list, patches would be accepted.  Closing this as we want to keep the queue a bit more manageable, but not objecting to this.
